pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -51,6 +50,21 @@ let
       hash = "sha256-bQC0QmkLalxdj4mDEdqvvOFtNwz2T1MpTDuMXGYeQ18=";
     };
   };
+
+  # Derived from subprojects/spdlog.wrap
```

Comment:
There should be a comment explaining why we choose to build these components from source. I didn't know about this and ignorantly added spdlog the normal way as a dependency.